### ISSUE TITLE
Cannot check deeplabcut version, "import deeplabcut" cannot execute

### ISSUE THREAD

**Issue by pranavgurusankar** [DeepLabCut/DeepLabCut#1413]
** System Information **
OS: MacOS 10.15.17
DeepLabCut Version: cannot even check this, terminal tells me "import: command not found" when I type "import deeplabcut"
Anaconda env used: DEEPLABCUT

**Describe the problem**
Cannot check deeplabcut version, and when I tried running the testscript I wasn't able to when I typed "pythonw testscript.py" [I am on MacOS]. I was otherwise able to install DLC properly, and activate the DEEPLABCUT environment as mentioned in the installation instructions. 

**Screenshots**
![Screenshot 2021-07-14 at 5 36 26 PM](https://user-images.githubusercontent.com/37655744/125710063-f96f1178-4b20-4ac8-8829-e7bd773645e3.png)

**Comment by jeylau** [comment]
Duplicate of #1218 #1307

pythonw points to a version of python that is different from the one of your environment, explaining why deeplabcut is not found. Execute `conda install python.app` and you're fine.

**Comment by pranavgurusankar** [comment]
Hi, I executed conda install python.app and it seemed to remove that error. I'm now able to run the testscript as per the tutorial video, but my output is quite different from the one shown in the video. I'm getting a lot of "exceptions" and quite a few errors. I've screenshotted a few of them and am showing them below:

![Screenshot 2021-07-20 at 4 22 40 PM](https://user-images.githubusercontent.com/37655744/126407463-51858222-1687-41c2-8f1e-b5a06f2af900.png)
![Screenshot 2021-07-20 at 4 23 15 PM](https://user-images.githubusercontent.com/37655744/126407466-dca430f5-381e-433e-aa32-9a3a3c730db2.png)

I also ran into some interesting messages during installation, not sure if this is relevant but I also screenshotted them, as seen below:

![Screenshot 2021-07-20 at 4 06 03 PM](https://user-images.githubusercontent.com/37655744/126407497-a49bda75-4de6-4ca6-bbe4-112a3565b778.png)
